spec/zoneddatetime.html

```diff
@@ -1321,6 +1321,43 @@ <h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-adddaystozoneddatetime" type="abstract operation">
+      <h1>
+        AddDaysToZonedDateTime (
+          _instant_: a Temporal.Instant,
+          _dateTime_: a Temporal.PlainDateTime,
+          _timeZone_: an Object,
+          _calendar_: an Object,
+          _days_: an integer,
+        ): either a normal completion containing a Record, or an abrupt completion
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          The returned Record has fields [[EpochNanoseconds]] (a BigInt), [[Instant]] (a Temporal.Instant), and [[DateTime]] (a Temporal.PlainDateTime). The _dateTime_ parameter should be the Temporal.PlainDateTime corresponding to _instant_ in _timeZone_.
```

Comment:
How do people feel about "should" where it's not in the RFC2119 sense in ecmaland?

Reply:
I agree this isn't ideal, but I couldn't think of another way to express it. "Assert: _dateTime_ is the value that would be returned from ! GetPlainDateTimeFor(_timeZone_, _instant_, _calendar_)" but that is not true for custom time zones because they could just return some random PlainDateTime on every call.

I think [GetBindingValue](https://tc39.es/ecma262/#sec-object-environment-records-getbindingvalue-n-s) might be a usage of "should" in ECMA-262 that is a good parallel to this: "The property should already exist but if it does not the result depends on _S_."

Here, _dateTime_ "should" be the result of GetPlainDateTimeFor, but if it isn't, the result depends on what _dateTime_ actually is.

Although now that I think about it, we could just sidestep the problem: "Assert: If _timeZone_ is a String, _dateTime_'s internal slots are the same as those of the Object returned from ! GetPlainDateTimeFor(_timeZone_, _instant_, _calendar_)." What do you think about this?

(Which also reminds me that the _timeZone_ and _calendar_ parameters in this operation's header can be strings, it's a mistake that they can only be objects.)

Reply:
Added some fixup commits with this assertions and the fixes to the _timeZone_ and _calendar_ parameters' types in the headers.

Reply:
Sounds good, thanks